tests/unit/DripVariableTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace yournamespace\tests;
+
+use Codeception\Test\Unit;
+use Craft;
+use extreme\drip\Drip;
+use Solspace\Freeform\Freeform;
+use UnitTester;
+
+class DripVariableTest extends Unit
+{
+  /**
+   * @var UnitTester
+   */
+    protected $tester;
+
+  /** @test * */
+    public function testDripAccountId()
+    {
+        $drip = new Drip('drip');
+        $drip->setSettings([
+        'dripAccountId' => '1234',
+        ]);
+        $drip->init();
+
+        $template = '{{ craft.drip.dripAccountId }}';
+
+        $output = Craft::$app->getView()->renderString($template);
+
+        $this->assertEquals('1234', $output);
+    }
+
+    /** @test
+     * Test that dripFields variable function returns array of at least 1 element
+     * dripFields returns value used to populate a select field
+     *
+     */
+
+    public function testDripFields()
+    {
+        $drip = new Drip('drip');
+        $drip->init();
+
+        $template = '{{ craft.drip.dripFields | length }}';
+
+        $output = (int) Craft::$app->getView()->renderString($template);
```

Comment:
What's the reasoning behind casting this to an integer? I know you have a test below, that checks for it

Reply:
have updated to simplify without additional int test

Reply:
Could you remove the (int) too from line 47?

Reply:
possibly. I was unsure whether assertGreaterThan would correctly handle the type conversion